packages/extension/src/components/DaDndMessage.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="dnd-message-bar">
+    <p class="do-not-disturb">Do not disturb.</p>
+    <a href="#" class="turn-off" role="button" @click="$emit('dndOff')">Turn Off</a>
+  </div>
+</template>
+
+<style scoped>
+.dnd-message-bar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 32px;
+  background-color: #7e48ff;
+  text-align: center;
+}
+
+.do-not-disturb {
+  font-style: oblique;
+}
+
+a.turn-off {
+  margin-left: 8px;
+  color: #fff;
+  font-size: 11px;
```

Comment:
Regarding typography styling we also have them predefined.
The right typography for you is called `nuggets`: https://github.com/dailynowco/daily-apps/blob/master/packages/components/src/styles/mixins/typography.pcss#L1
You can either add the `nuggets` class to your element or use the mixin `@mixin nuggets;` (I think in this case it's better to use class)